content/en/docs/concepts/storage/volumes.md

```diff
@@ -12,10 +12,12 @@ weight: 10
 <!-- overview -->
 
 On-disk files in a container are ephemeral, which presents some problems for
-non-trivial applications when running in containers. One problem
-is the loss of files when a container crashes. The kubelet restarts the container
-but with a clean state. A second problem occurs when sharing files
-between containers running together in a `Pod`.
+non-trivial applications when running in containers. First problem occurs when 
```

Comment:
Some rewording suggestions:

```
One problem occurs when 
a container crashes or is stopped. Container state is not saved so all of the files that were created or modified during the lifetime of the container are lost. During a crash, kubelet restarts the container with 
a clean state. Another problem occurs when multiple containers are running in 
a `Pod` and need to share files. It can be challenging to setup and access a shared filesystem across all of the containers.
```

Reply:
suggested changes done.